.travis.yml

```diff
@@ -18,16 +18,16 @@ services:
 install:
 - docker pull ${LEARNOCAML_SERVER_IMAGE}
 - docker pull ${LEARNOCAML_CLIENT_IMAGE} 
-- docker run -d -i --name=LEARNOCAML_SERVER -v learn-ocaml-sync-demo:/sync -v $PWD/pfitaxel-server/src:/repository:ro ${LEARNOCAML_SERVER_IMAGE}
-- docker run -d -i --link LEARNOCAML_SERVER:LEARNOCAML_SERVER --name=LEARNOCAML_CLIENT -v ${TRAVIS_BUILD_DIR}/tests.sh:/tests.sh ${LEARNOCAML_CLIENT_IMAGE}
+- docker network create learnocaml
+- docker run -d --net=learnocaml --name=LEARNOCAML_SERVER -v $PWD/test-directory/pfitaxel-server/src:/repository:ro ${LEARNOCAML_SERVER_IMAGE}
```

Comment:
Dear @leunam217, a very last comment on the naming of the containers:

for the images, `LEARNOCAML_SERVER_IMAGE` is very OK, but but it is possible that `--name=LEARNOCAML_SERVER` doesn't fully work because the hostname of the container (in the `learnocaml` network) must have letters/digits/hyphen, not underscore.

Hence this last proposal for the container names:
```
docker run -d --net=learnocaml --name=learnocaml-server
docker run -d --net=learnocaml --name=learnocaml-client
```

Reply:
i was not understanding why it didn't work thank you